pom.xml

```diff
@@ -3353,11 +3353,6 @@
       <!-- Default hadoop profile. Uses global properties. -->
     </profile>
 
-    <profile>
```

Comment:
We could possibly leave the profile in and have it still do nothing, as if the profile doesn't exist, a build with -Phive-2.3 fails, even though it's really "OK" - it's already hive 2.3 by default. I don't feel strongly about it.

Reply:
> We could possibly leave the profile in and have it still do nothing, as if the profile doesn't exist, a build with -Phive-2.3 fails, even though it's really "OK" - it's already hive 2.3 by default. I don't feel strongly about it.

you mean `test-hive2.3`? I think this won't work now and we don't need this too.

Reply:
No, I just mean do not remove the profile, so that specifying it does not fail the build

Reply:
> No, I just mean do not remove the profile, so that specifying it does not fail the build

But build with a `-Phive-2.3` won't failed too...

Reply:
After you remove this? Won't maven say that the profile doesn't exist?

Reply:
yea, won't failed with a profile doesn't exist, So I make this pr.  I know your concern.

Reply:
Can you mention this in the PR description? Saying that -Phive-2.3 still works after removing the profile

Reply:
> 

Done